README.md

```diff
@@ -1,121 +1,115 @@
+
+```python3
+>>> from tslearn.preprocessing import TimeSeriesScalerMinMax
+>>> from tslearn.piecewise import SymbolicAggregateApproximation
+>>> X_norm = TimeSeriesScalerMinMax().fit_transform(X)
+>>> X_sax = SymbolicAggregateApproximation(4, 10).fit_transform(X_norm)
+>>> print(X_sax)
+[[[5] [7] [8] [6]] 
+ [[5] [6] [8] [6]]
+ [[5] [6] [8] [6]]]
+```
 
-### Pre-requisites
+## 3. Training a model
 
-When using PyPI, C++ build tools should be available to perform installation.
+After getting the data in the right format, a model can be trained. Depending on the use case, tslearn supports different tasks: classification, clustering and regression. For an extensive overview of possibilities, check out our [gallery of examples](https://tslearn.readthedocs.io/en/latest/auto_examples/index.html).
 
-Using `pip` should also work fine:
-```bash
-python -m pip install tslearn
+```python3
+>>> from tslearn.shapelets import ShapeletModel
+>>> clf = ShapeletModel({3: 1})
+>>> clf.fit(X_sax, y)
+>>> print(clf.shapelets_)
+[[[4.8] [8] [6]]]
 ```
 
-In this case, you should have `numpy`, `cython` and C++ build tools available
-at build time.
 
-## Using latest github-hosted version
+## 4. Evaluation and analysis
 
-If you want to get `tslearn`'s latest version, you can refer to the repository
-hosted at github:
-```bash
-python -m pip install https://github.com/rtavenar/tslearn/archive/master.zip
-```
+tslearn further allows to perform all different types of analysis. Examples include [calculating barycenters](https://tslearn.readthedocs.io/en/latest/gen_modules/tslearn.barycenters.html#module-tslearn.barycenters) of a group of time series or calculate the distances between time series using a [variety of distance metrics](https://tslearn.readthedocs.io/en/latest/gen_modules/tslearn.metrics.html#module-tslearn.metrics).
 
-## Troubleshooting
+# Documentation
 
-It seems on some platforms `Cython` dependency does not install properly.
-If you experiment such an issue, try installing it with the following command:
+The documentation, including a gallery of examples, is hosted at [readthedocs](http://tslearn.readthedocs.io/en/latest/index.html).
 
-```bash
-python -m pip install cython
-```
+# Contributing
+
+If you would like to contribute to `tslearn`, please have a look at [our contribution guidelines](CONTRIBUTING.md). A list of interesting TODO's can be found [here](https://github.com/rtavenar/tslearn/issues?utf8=✓&q=is%3Aissue%20is%3Aopen%20label%3A%22new%20feature%22%20). **If you want other ML methods for time series to be added to this TODO list, do not hesitate to open an issue!**
```

Comment:
Here, maybe we could add a link to the "new issue" page: https://github.com/rtavenar/tslearn/issues/new/choose